test/v3/USTBv3.t.sol

```diff
@@ -0,0 +1,94 @@
+pragma solidity ^0.8.28;
+
+import "forge-std/StdUtils.sol";
+import {Test} from "forge-std/Test.sol";
+import {console} from "forge-std/console.sol";
+import {PausableUpgradeable} from "openzeppelin-contracts-upgradeable/security/PausableUpgradeable.sol";
+
+import "openzeppelin-contracts/contracts/proxy/transparent/TransparentUpgradeableProxy.sol";
+import "openzeppelin-contracts/contracts/proxy/transparent/ProxyAdmin.sol";
+
+import {SuperstateTokenV2} from "src/v2/SuperstateTokenV2.sol";
+import {USTBv2} from "src/v2/USTBv2.sol";
+import {USTB} from "src/USTB.sol";
+import {AllowList} from "src/AllowList.sol";
+import "test/SuperstateTokenTestBase.t.sol";
+
+contract USTBv3Test is SuperstateTokenTestBase {
+    SuperstateTokenV1 public tokenV1;
+    SuperstateTokenV2 public tokenV2;
+
+    function setUp() public override {
```

Comment:
Think this test is not initializing v3

Reply:
There's nothing that needs to be initialized for v3